pytest_reportportal/listener.py

```diff
@@ -53,67 +64,83 @@ def pytest_runtest_makereport(self, item):
                 loglevel='ERROR',
             )
 
+        # Defining test result
         if report.when == 'setup':
             self.result = None
             self.issue = {}
-            if report.failed:
-                # This happens for example when a fixture fails to run
-                # causing the test to error
+
+        if report.failed:
+            if (self.result in (None, 'PASSED', 'SKIPPED')):
                 self.result = 'FAILED'
-                self._add_issue_info(item, report)
-            elif report.skipped:
-                # This happens when a testcase is marked "skip".  It will
-                # show in reportportal as not requiring investigation.
+        elif report.skipped:
+            if (self.result in (None, 'PASSED')):
                 self.result = 'SKIPPED'
-                self._add_issue_info(item, report)
+        else:
+            if (self.result is None):
+                self.result = 'PASSED'
 
-        if report.when == 'call':
```

Comment:
Why do not we take into account this runtest phase anymore?

Reply:
We  just applying or keeping the most bad result (pass -> skip -> fail) and phase doesn't mater

Reply:
Got it, thanks.